File: Auxiliar 1 - 2019P/kotlin/cc3002/auxiliares2019p/aux1/Main.kt
package cc3002.auxiliares2019p.aux1


/**
 * Simple example of using this package classes.
 */
fun main() {
    val lyn = Hero("Lyn")
    val eliwood = Hero("Eliwood")
    println("${lyn.name} has ${lyn.hitPoints} hit points.")
    println("${eliwood.name} has ${eliwood.hitPoints} hit points.")
    lyn.attack(eliwood)
    println("${lyn.name} attacked ${eliwood.name}")
    println("${eliwood.name} has ${eliwood.hitPoints} hit points.")
}

File: Auxiliar 1 - 2019P/python/cc3002/auxiliares2019p/aux1/main.py
from cc3002.auxiliares2019p.aux1.hero import Hero

if __name__ == '__main__':
    lyn = Hero("Lyn")
    eliwood = Hero("Eliwood")
    print(f"{lyn.name} has {lyn.hit_points} hit points")
    print(f"{eliwood.name} has {eliwood.hit_points} hit points")
    lyn.attack(eliwood)
    print(f"{lyn.name} attacked {eliwood.name}")
    print(f"{eliwood.name} has {eliwood.hit_points} hit points")
